tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -827,6 +829,24 @@ public void testNoAcceptHeader() {
     }
 
 
+    @Test
+    @RunAsClient
+    @InSequence(41)
+    public void testCustomUnitAppendToGaugeName() {
+        Header wantPrometheusFormat = new Header("Accept", TEXT_PLAIN);
+        given().header(wantPrometheusFormat).get("/metrics/application").then().statusCode(200)
+        .and().body(containsString("TYPE application:org_eclipse_microprofile_metrics_test_metric_app_bean_gauge_me_b_hands gauge"));
+    }
+
+    @Test
+    @RunAsClient
+    @InSequence(42)
+    public void testCustomUnitForCounter() {
+        Header wantPrometheusFormat = new Header("Accept", TEXT_PLAIN);
+        given().header(wantPrometheusFormat).get("/metrics/application").then().statusCode(200)
+        .and().body(containsString("TYPE application:org_eclipse_microprofile_metrics_test_metric_app_bean_count_me_b counter"));
```

Comment:
Do we need a _total at the end?

Reply:
Hello, are you asking if the output at the metrics endpoint contains _total, or if we should make the output include _total?

Reply:
SmallRye implementation includes the unit name in this case, I think that's wrong according to the spec
And also, you have specified `absolute=true` for this metric, so I'm getting this:
`TYPE application_metric_test_test1_count_me_b_jellybean counter`

I think we should be getting
`TYPE application_metric_test_test1_count_me_b_total counter`

Reply:
Ah so I noticed that in SmallRye this is already getting fixed by PR https://github.com/smallrye/smallrye-metrics/pull/31/files

Reply:
@jmartisk / @pilhuhn , I agree it should be `TYPE application_metric_test_test1_count_me_b_total counter` since the `_total` is required in the master branch now.

Reply:
@Erikkre , need to update line 847 to have _total at the end of the metric name